R/tostring.R

```diff
@@ -103,24 +105,78 @@ setMethod("toString", "MatrixPrintForm", function(x,
                                                   tf_wrap = FALSE,
                                                   max_width = NULL,
                                                   col_gap = x$col_gap,
-                                                  hsep = default_hsep()) {
-  mat <- matrix_form(x, TRUE)
+                                                  hsep = default_hsep(),
+                                                  indent_size = NULL) {
+
+  checkmate::assert_integerish(indent_size, null.ok = TRUE)
+  checkmate::assert_flag(tf_wrap)
+
+  mat <- matrix_form(x, indent_rownames = TRUE) # Fix for indent_size should be here
   inset <- table_inset(mat)
 
+  # Standard indentation size and inserted one
```

Comment:
The following is a tentative reparation for the case in which indent_size is set only here and not before. There is something about widths and ncchar that is not calculated correctly afterward, so even if the indentation is reconstructed correctly it produces a wider table at the end. See related test in rtables.

Reply:
indent size shouldn't be set for the `toString MatrixPrintForm` method, that should always use the indent_size on the MatrixPrintForm object. indent_size is set (on the `MatrixPrintForm` object) for the `toString VTableTree` method in rtables, because rtables doesn't have a concept of indent size in the object (as that is an aspect of rendering). Sorry that was not clear from my previous comment.

Reply:
Thank you! Now it is clear. I got confused by the fact that `toString` is both in rtables and formatters eheh. It makes sense now! Reverted these changes